tradeit-android-sdk/src/main/java/it/trade/android/sdk/enums/TradeItOrderAction.java

```diff
@@ -1,10 +1,9 @@
 package it.trade.android.sdk.enums;
 
-
 public enum TradeItOrderAction {
     BUY("buy"),
     SELL("sell"),
-    BUY_TO_COVER("buyTocover"),
+    BUY_TO_COVER("buyToCover"),
```

Comment:
This seems like it was a typo in the value?

Reply:
👍